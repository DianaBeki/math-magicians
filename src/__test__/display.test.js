import { render } from '@testing-library/react';
import FetchQuotes from '../components/display';

  <render />;
describe('testing fetch quotes', () => {
  test('testing snapshots', () => {
    const tree = render.create(<FetchQuotes />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
