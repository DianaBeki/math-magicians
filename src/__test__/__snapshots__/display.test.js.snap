// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing fetch quotes testing snapshots 1`] = `[Function]`;
